scripts/update_specs_services_from_ssm.py

```diff
@@ -24,7 +24,7 @@
 service_map = {
     'acm': ['AWS::CertificateManager::'],
     'apigateway': ['AWS::ApiGateway::', 'AWS::ApiGatewayV2::'],
-    'application-autoscaling': ['AWS::ApplicationAutoScaling::'],
+    # 'application-autoscaling': ['AWS::ApplicationAutoScaling::'], ## remove because SSM endpoints aren't correct
```

Comment:
how do we handle these long term? does this happen often? is there someone we can reach out to about these? are they usually fixed quickly?

Reply:
This is the first time I have noticed an issue with this data and I'm not sure where to take it yet.

Reply:
Submitted a support ticket.  I'll update the script once the issue is resolved.